META.json

```diff
@@ -27,6 +27,9 @@
       "configure" : {
          "requires" : {
             "ExtUtils::MakeMaker" : "0"
+         },
+         "suggests" : {
+            "JSON::PP" : "2.27300"
```

Comment:
Where did this suggestion come from?

Reply:
this after running the `localdzil`

Reply:
We can probably drop it then, since the provided modules don't specifically use it.